pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -168,7 +168,7 @@ def work(self):
                                             '[#] {} has been exchanged for candy!'.format(pokemon_name), 'green')
                                     else:
                                         logger.log(
-                                        '[x] {} has been added to your Pokémon! [CP {}]'.format(pokemon_name, cp), 'green')
+                                        '[x] {} has been added to your Pokémon!'.format(pokemon_name, cp), 'green')
```

Comment:
I suggest you either to bring the `[CP {}]` part back into the message or to remove the `cp` argument from the `.format(pokemon_name, cp)` part.